codemirror/codemirror.d.ts

```diff
@@ -786,8 +786,8 @@ declare module CodeMirror {
         and thus the browser's text search works on it. This will have bad effects on performance of big documents. */
         viewportMargin?: number;
 
-        /** Optional lint configuration to be used in conjunction with CodeMirror's linter addon. */
-        lint?: LintOptions;
+        /** LintOptions|boolean. Optional lint configuration to be used in conjunction with CodeMirror's linter addon. */
+        lint?: any;

```

Comment:
that's fine with me, but the other places in the typing which support multiple types use 'any' as well. Union types only showed up in TS 1.4 (https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#union-types), so I assumed we're shooting for deeper backcompat. Would you prefer I change it?